project/Dependencies.scala

```diff
@@ -34,28 +36,15 @@ object Dependencies {
     "net.bytebuddy" % "byte-buddy-agent" % "1.10.18"
   )
 
-
   val logback = Seq("ch.qos.logback" % "logback-classic" % LogbackVersion)
 
   val openTelemetryApi = Seq(
-    "io.opentelemetry" % "opentelemetry-api" % "0.13.0"
-  )
-
-  val openTelemetrySdk = Seq(
-    "io.opentelemetry" % "opentelemetry-sdk" % "0.13.0"
+    "io.opentelemetry" % "opentelemetry-api" % OpentelemetryVersion
   )
 
-  val openTelemetryDependenciesOverrides = Seq(
-    "io.opentelemetry" % "opentelemetry-api-metrics" % "0.13.0-alpha",
-    "io.opentelemetry" % "opentelemetry-sdk-metrics" % "0.13.0-alpha"
-  )
-
-  val newRelicSdk = Seq(
-    "com.newrelic.telemetry" % "opentelemetry-exporters-newrelic" % "0.13.1",
-    // TODO uncomment the line above when solve this issue: https://github.com/newrelic/opentelemetry-exporter-java/issues/149
-    // For a while, we use it as a unmanaged dependency at extension/lib
-    "com.newrelic.telemetry" % "telemetry"             % "0.10.0",
-    "com.newrelic.telemetry" % "telemetry-http-okhttp" % "0.10.0"
+  val openTelemetryMetrics = Seq(
+    "io.opentelemetry" % "opentelemetry-api-metrics" % OpentelemetryMetricsVersion,
```

Comment:
Do you think it would be a good idea to use classifiers here? So this would become :
`"io.opentelemetry" % "opentelemetry-api-metrics" % OpentelemetryVersion classifier "alpha"`

Reply:
I don't have an opinion. It doesn't really solve anything. And I'm not sure how Scala Steward looks at it.

Reply:
You're right, this doesn't seem to solve anything.